virtcontainers/qemu.go

```diff
@@ -763,6 +763,12 @@ func (q *qemu) startSandbox(timeout int) error {
 	var strErr string
 	strErr, err = govmmQemu.LaunchQemu(q.qemuConfig, newQMPLogger())
 	if err != nil {
+		if q.config.Debug && q.qemuConfig.LogFile != "" {
+			b, err := ioutil.ReadFile(q.qemuConfig.LogFile)
```

Comment:
I think here you are overwriting the previous error, isn't it? you should use a different name for `err`

Reply:
Hi @devimc , yes, here it overwriting the previous error only in its scope, and it will not
affect the following "err" which is out of its scope.